crates/wast/src/ast/expr.rs

```diff
@@ -411,7 +411,7 @@ instructions! {
         TableSize(TableArg<'a>) : [0xfc, 0x10] : "table.size",
         TableGrow(TableArg<'a>) : [0xfc, 0x0f] : "table.grow",
 
-        RefNull(RefType<'a>) : [0xd0] : "ref.null",
+        RefNull(HeapType<'a>) : [0xd0] : "ref.null",
```

Comment:
Should this be removed to account for https://github.com/WebAssembly/reference-types/pull/100?

Reply:
Ah no, that just applies to ref.is_null which was #28. This change is from function-references [1].

[1] https://github.com/WebAssembly/function-references/blob/master/proposals/function-references/Overview.md#optional-references

Reply:
Wow apparently I need to learn how to read...